### PR TITLE
MRG: Link picking informations in _LinkViewer

### PR DESCRIPTION
**Pull request by GuillaumeFavelier**
This PR links picking points in `_LinkViewer` and adds a `picking` parameter to `link_brains()`.

- [x] Link `add_point()`
- [x] Link `remove_point()`
- [x] Synchronize points colors *(mitigated)*
- [x] Link the initial picked points

Related to https://github.com/mne-tools/mne-python/issues/7959#issue-650610319
It's an item of https://github.com/mne-tools/mne-python/issues/7162

### CODE REVIEW

File: mne/viz/_3d.py [hunk 1]
```diff
@@ -1516,7 +1516,7 @@ def _plot_mpl_stc(stc, subject=None, surface='inflated', hemi='lh',
     return fig
 
 
-def link_brains(brains, time=True, camera=False):
+def link_brains(brains, time=True, camera=False, picking=False):
```
Comment (hoechenberger) [inline mne/viz/_3d.py:1519]: the `picking` parameter need to be documented

**Comment by GuillaumeFavelier** [comment]
I mitigated the issue with synchronized colors by just clearing the initial picked points. I don't have a real solution for now.

**Comment by larsoner** [comment]
Show a union of all picked points on all linked brains at the time of linking, by removing then re-adding once linked? That way at least the first brain will have colors stay the same, and all picked points persist

**Comment by GuillaumeFavelier** [comment]
I'll build the union list and re-add then

**Comment by agramfort** [comment]
thx @GuillaumeFavelier

**Pull request merged**